src/commands/start.yml

```diff
@@ -15,10 +15,8 @@ parameters:
     description: Whether to install the `awslocal` command line interface into the build environment (requires python3 and pip3)
 
 steps:
-  - run:
-      name: Install LocalStack
-      command: |
-        test "<<parameters.install-awslocal>>" != "true" || pip3 install -q localstack awscli-local[ver1]
+  - tools:
+      install_awslocal: << parameters.install-awslocal >>
```

Comment:
I think this might have already been the case in previous versions of the orb, but to me it seems that the start command does not work with `install-awslocal==false` because this step actually installs the LocalStack CLI (only if `install-awslocal==true`) which is then used in the start step.

Reply:
This is a nice catch that we've overlooked in the past a couple of times as I can tell. 🥇